hack/build-images.sh

```diff
@@ -7,7 +7,7 @@ set -o pipefail
 STARTTIME=$(date +%s)
 source_root=$(dirname "${0}")/..
 
-prefix="openshift/origin-"
+prefix="docker.io/openshift/origin-"
```

Comment:
We really should not be specifying docker.io here.

Why is this required? If the build needs this, then it should pass it as a value.